$NetBSD: patch-Makefile,v 1.2 2026/09/24 15:14:28 wiz Exp $

Do not override CFLAGS/LDFLAGS.
Pass all LDFLAGS to make subprocesses.

--- Makefile.orig	2020-11-06 01:13:09.000000000 +0000
+++ Makefile
@@ -71,14 +71,12 @@ GCFLAG=
 #GCFLAG=-D_ATS_GCBDW
 #GCFLAG=-D_ATS_GCATS
 #
-CFLAGS=
-LDFLAGS=
 #
 ######
 #
 src2_patsopt: ; \
 $(MAKEJ4) -C src/CBOOT patsopt \
-CCOMP=$(CCOMP) GCFLAG=$(GCFLAG) LDFLAGS=$(LDFLAGS) 
+CCOMP=$(CCOMP) GCFLAG=$(GCFLAG) LDFLAGS='$(LDFLAGS)'
 #
 bin2_patsopt: src2_patsopt ; $(CPF) src/CBOOT/patsopt bin/patsopt
 #
